package.json

```diff
@@ -15,16 +15,34 @@
       "react"
     ]
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.{js,jsx}": [
+      "pretty-quick --staged",
+      "eslint . --fix",
+      "git add"
```

Comment:
Why the `git add` here?

Reply:
@popojargo Excellent question. Its because if eslint produces any fixes it will pipe in those files through `git add` automatically.